.projenrc.js

```diff
@@ -26,8 +26,7 @@ const project = new web.ReactTypeScriptProject({
     },
   },
 
-  minNodeVersion: "12.20.0",
-  workflowNodeVersion: "12.22.0",
+  minNodeVersion: "16.0.0",
```

Comment:
`react-scripts` requires node 14.0.0, but `eslint` requires:

```console
eslint@8.5.0: The engine "node" is incompatible with this module. Expected version "^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.15.4"
```

Figured we just bump to 16 and be done with it.